osu.Game/Screens/Play/HUD/HitErrorMeters/BarHitErrorMeter.cs

```diff
@@ -273,45 +274,74 @@ private void recreateLabels(LabelStyles style)
                     break;
 
                 case LabelStyles.Icons:
-                    labelEarly = new SpriteIcon
+                    labelEarly = new UprightAspectMaintainingContainer
```

Comment:
The fact that this is created in four places within a switch statement puts me off a bit. The cleaner/better way to do this, in my opinion, would be to create the `UprightAspectMaintainingContainer`s in BDL, add them once, and only clear or replace their children in `recreateLabels()`, rather than recreate the entire container every time the setting changes.